contracts/QuestFactory.sol

```diff
@@ -473,22 +489,26 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         return ECDSAUpgradeable.recover(messageDigest, signature_);
     }
 
+    function claimRewards(string memory questId_, bytes32 hash_, bytes memory signature_) external payable nonReentrant sufficientMintFee claimChecks(questId_, hash_, signature_) {
```

Comment:
Is there a mint fee applied for Quest contract? Seems like default would be 0, but curious about why there's a mint fee for the erc20 contract